src/azure-cli/azure/cli/command_modules/keyvault/_format.py

```diff
@@ -0,0 +1,41 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+"""Table transformer for keyvault commands"""
+
+
+def build_table_output(result, projection):
+
+    if not isinstance(result, list):
+        result = [result]
+
+    final_list = []
+
+    from collections import OrderedDict
+    for item in result:
+        def _value_from_path(each_item, path):
+            obj = each_item
+            try:
+                for part in path.split('.'):
+                    obj = obj.get(part, None)
+            except AttributeError:
+                obj = None
+            return obj or ' '
+
+        item_dict = OrderedDict()
+        for element in projection:
+            item_dict[element[0]] = _value_from_path(item, element[1])
+        final_list.append(item_dict)
+
+    return final_list
```

Comment:
This function already appears at

https://github.com/Azure/azure-cli/blob/c3fd340dbd8a734c6246bf0f860539d0fb71bca8/src/azure-cli/azure/cli/command_modules/storage/_format.py#L13-L36

Better to extract it to core to avoid `duplicate-code` which we explicitly disabled because we have no time to fix

https://github.com/Azure/azure-cli/blob/ed6e829554624894d5c94ad71bb85e788d726a43/pylintrc#L18

Reply:
extracted